src/routes/docs/products/functions/functions/+page.markdoc

```diff
@@ -0,0 +1,185 @@
+---
+layout: article
+title: Functions
+description: Learn what an Appwrite Function can do for you and how to create a new Appwrite Function
+---
+
+Each Appwrite Function is a piece of user defined code that can be executed on demand.
```

Comment:
I wouldnt call it user-defined. either deveoper-defined, or defined by you.
User-defined sounds like end user

Reply:
Addressed